### ISSUE TITLE
Syncthing still doesn't auto-start in Fedora 27

### ISSUE THREAD

**Issue by tristanbrown** [syncthing/syncthing-gtk#448]
Ongoing issue from: https://github.com/syncthing/syncthing-gtk/issues/444
https://github.com/syncthing/syncthing-gtk/pull/446 fixed the issue with the checkbox, but the app still doesn't start automatically. So, the "Start Syncthing-GTK when I login to my desktop" box remains checked now, but the program doesn't start on login. 

My `.config/autostart/Syncthing-GTK.desktop` file is exactly as shown in https://github.com/syncthing/syncthing-gtk/issues/444: 
```
[Desktop Entry]
Name=Syncthing-GTK
Exec=/usr/bin/syncthing-gtk
Icon=/usr/share/syncthing-gtk/icons/st-logo-128.png
Comment=GUI for Syncthing
X-GNOME-Autostart-enabled=true
Hidden=false
Type=Application
```

**Comment by kozec** [comment]
I don't see anything specific that could prevent it from work... Does it shows in "Session and startup" applet in Gnome settings? Are you running Gnome on Wayland or is there anything else special about your setup?

**Comment by tristanbrown** [comment]
No, it's not showing up in the startup applications in Gnome settings. I can add it, but that state is not saved.

**Comment by kozec** [comment]
> I can add it, but that state is not saved.

How exactly are you adding and checking? I have VM with Fedora 27 here and gnome-tweak-tool works as expected, both when adding and when displaying what ST-GTK added, although it displays wrong icon in later case.

**Comment by tristanbrown** [comment]
Using the GUI for Gnome Tweaks.

**Comment by kozec** [comment]
So, you just go to gnome tweak, autostart applications, click on plus, add ST-GTK and it doesn't stick there?

I'm almost sure that's not problem with ST-GTK...

**Comment by decathorpe** [comment]
@tristanbrown Are you using the TopIcons plus extension for GNOME for displaying legacy tray icons?

**Comment by xtramon** [comment]
Same problem with KDE on Arch Linux

**Comment by Zettt** [comment]
I'd like to report the same issue for Raspbian. The .desktop entry is created correct, but the app won't autostart. Just an `ls` on the autostart folder gives a wrong permission error. Do I have to chown this? Here's a post on the [Raspberry forum](https://www.raspberrypi.org/forums/viewtopic.php?t=132637&__cf_chl_captcha_tk__=078841f2370abcb55d9bd30fac99f42058cd822d-1585202627-0-AcHm-5ftguAShn-bKX52tkdzo2XyZK5Ux6xyM5GNAtm2K0lnfmgqvNIdpyIUeNefUs9Ugr3A-uTSXPV5BQge6wyL8jG1-QlYU_0OqDx5Mfp6cPG3Nb5bKKd9m_K4PNeIAKrWjgsGzMSkMVrl_QID2zX9Pg3VKaaZr001Jimww4msAF2RbMEOxmE96on8HACvVvGQXbxeDkr8U5GHdy-_yWGJIN1n58XIBWzbsE9-2_tMr4iY95Zqcu5IX4DTH16_aY5RXcv1pQQpvUbMCGU__eF7aseTZYqnHb65PFGun0UbAZ4bhtWL4H_jQ9go6fI7dhfOSHb9e_OrTyY3hFjOtVyo1L1qkONXmt3ZLzCKE-R2lrAF5S2-9bUX2RbMOqk-xOgg3ffXi1ANXGS9MplWqaU). I'm not super skilled with Linux to figure out what the best route of action is here. Please excuse.

**Comment by sophiekuehn** [comment]
On Mint 19.3 Cinnamon, the -desktop file is not added at all.
